highfidelity/hifi
# [DNM] Game table

## QA Test
Use the script at: https://raw.githubusercontent.com/thoys/hifi/gameTable/unpublishedScripts/marketplace/gameTable/createGameTable.js to create the gameTable.
This test requires at least two persons.
1. Press the ">>" button to setup the first game.
2. Have the other user press >> or reload, and switch around a couple of times.
3. The board picture and the pieces should match, e.g. Chessboard for Chess, Checkers board for Checkers.
4. Rotate the table in a different angle, the grid based board games should still have their pieces in the correct rotation, e.g. the chessboard knights should point towards the center.

## Dev note:
[DO NOT MERGE] this PR contains a lot of assets, we want to move this to a different repository eventually.
